### PR TITLE
Embedding echosign docs with iframes

### PR DESCRIPTION
**Pull request by rhyolight**
Addresses:
- https://jira.numenta.com/browse/WEB-472
- https://jira.numenta.com/browse/WEB-474
- https://jira.numenta.com/browse/WEB-476

Added better instructional text on both pages with embeddings,
explaining what to do if there is no Flash, and how to get to the direct
signing page if the embedding doesn't load.

Switched from javascript embed (not loading on localhost or staging) to
an iframe embed, which loaded in all environments.

### CODE REVIEW

File: packages/numenta.org/pages/licenses/contrib.md [hunk 1]
```diff
@@ -9,16 +9,18 @@ Algorithm (CLA) referenced in our source code and documentation.
 
 ### Please read and sign the Numenta Contributor License below:
 
-*Use your **github username** as the optional "Public Name" value.*
+> Use your **github username** as the optional "Public Name" value.
 
 ---
 
-<script src="https://secure.echosign.com/public/embeddedWidget?wid=CBFCIBAA2AAABLblqZhB5rcsG-E9OX4keHHTob72yY2GZtKetGhXxARrVNTTlcPF1Tu9IOlFnCGXNyOhZ96U*"></script>
+<iframe src="https://secure.na1.echosign.com/public/esignWidget?wid=CBFCIBAA3AAABLblqZhBlxAtRy60dQbf-3OV5_q8O6JzRbnSnmBDMhphe0X2-nxXxCddVIAgu2Y1Cf_0A-eU*&hosted=false" width="100%" height="100%" frameborder="0" style="border: 0; overflow: hidden; min-height: 500px; min-width: 600px;"></iframe>
```
Comment (rhyolight) [inline packages/numenta.org/pages/licenses/contrib.md]: I realize the widget id changed, and I verified that it is still the correct document. This was a change on the Adobe side.

File: packages/numenta.org/pages/licenses/trial.md [hunk 2]
```diff
@@ -2,16 +2,19 @@
...
 
+Work under a temporary license and create a proof-of-concept and decide later whether to convert to a commercial license or go AGPL.
+
 ### Please read and sign the Numenta Trial License document below:
 
 ---
 
-<script src="https://secure.echosign.com/public/embeddedWidget?wid=CBFCIBAA2AAABLblqZhBLTxUqDnhed_-yFSwUwBB72hIfRdvbhCdJVqPi5Ai-HWridyTAheYtHZyRDWq7uwA*"></script>
-
+<iframe src="https://secure.na1.echosign.com/public/esignWidget?wid=CBFCIBAA3AAABLblqZhA1E2nxUTpvO8y8Zm3cR_4xsyDjIHywL_EyF1i1FcezAW7F2ATd7wbxJlBRgwivwsg*&hosted=false" width="100%" height="100%" frameborder="0" style="border: 0; overflow: hidden; min-height: 500px; min-width: 600px;"></iframe>
```
Comment (rhyolight) [inline packages/numenta.org/pages/licenses/trial.md]: I verified the embedded document was correct even with the id change here as well.

**Comment by rhyolight** [comment]
@lscheinkman please review 
cc @cbaranski @cmaver

**Comment by lscheinkman** [comment]
👍 LGTM

**Pull request merged**